src/new-components/button-link/button-link.test.tsx

```diff
@@ -217,6 +217,27 @@ describe('ButtonLink', () => {
         expect(link).not.toHaveClass('size-small')
     })
 
+    it('applies different class names based on width and alignment', () => {
+        render(
+            <ButtonLink href="/" variant="primary" width="full" align="right">
+                Click me
+            </ButtonLink>,
+        )
+        const buttonLink = screen.getByRole('link', { name: 'Click me' })
+        expect(buttonLink).toHaveClass('align-right')
+        expect(buttonLink).toHaveClass('width-full')
+    })
+
+    it('ignores align when width is not full', () => {
+        render(
+            <ButtonLink href="/" variant="primary" align="right">
```

Comment:
Would be great if this can trigger `@ts-expect-error`

Reply:
Done